minecraft-codev-forge/src/main/kotlin/net/msrandom/minecraftcodev/forge/MinecraftCodevForgePlugin.kt

```diff
@@ -19,6 +23,10 @@ open class MinecraftCodevForgePlugin<T : PluginAware> : Plugin<T> {
         applyPlugin(target) {
             createSourceSetConfigurations(PATCHES_CONFIGURATION)
 
+            createSourceSetElements {
+                dependencies.add(it.runtimeOnlyConfigurationName, "net.msrandom:codev-forge-runtime:0.1.0")
+            }
+
```

Comment:
Is this the place to add the dependency?

Reply:
Do not add the dependency to every source set. This should probably be left to cloche, as it knows which source sets need it while codev does not

Reply:
https://github.com/terrarium-earth/cloche/pull/57/commits/6c284d43453b3ff4e288946dd42ac73111e37198